fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/FromAdapter.java

```diff
@@ -45,13 +47,71 @@ public FromAdapter from(String tableName, Alias alias) {
         return this;
     }
 
+    public FromAdapter innerJoin(String tableName, Alias alias, WhereFragment joinOnPredicate) {
+        this.select.addInnerJoin(tableName, alias, joinOnPredicate.getExpression());
+        return this;
+    }
+
+    public FromAdapter leftOuterJoin(String tableName, Alias alias, WhereFragment joinOnPredicate) {
+        this.select.addLeftOuterJoin(tableName, alias, joinOnPredicate.getExpression());
+        return this;
+    }
+
+    public FromAdapter innerJoin(Select subQuery, Alias alias) {
```

Comment:
what does subquery and alias do? I'm really not sure what this method does

Reply:
nothing. Removed